packages/dapp/src/components/utility/TransactionStatusModalsHOC.tsx

```diff
@@ -153,10 +153,12 @@ export const hasTransactionStatusModals = (transactionStatusModalConfig: Transac
         (multiStepTransactionLabels && multiStepTransactionLabels[this.state.transactionType!]) || "1 of 1";
       const stepLabel = `Step ${stepLabelText} - ${transactionLabel}`;
       return (
-        <MetaMaskModal ipfsPost={true} waiting={true}>
-          <ModalStepLabel>{stepLabel}</ModalStepLabel>
-          <ModalHeading>Posting to IPFS</ModalHeading>
-        </MetaMaskModal>
+        <Modal width={558} textAlign="center">
+          <ProgressModalContentInProgress>
```

Comment:
this modal would give inaccurate info about what's happening - we could create a new container but it's not correct to say there's a tx happening here